Core/Core/Inbox/AttachmentPicker/ViewModel/AttachmentPickerViewModel.swift

```diff
@@ -29,26 +29,41 @@ class AttachmentPickerViewModel: ObservableObject {
     @Published public var isImagePickerVisible: Bool = false
     @Published public var isTakePhotoVisible: Bool = false
     @Published public var isAudioRecordVisible: Bool = false
+    @Published public var isFileSelectVisible: Bool = false
     @Published public private(set) var fileList: [File] = []
+    @Published public private(set) var alreadyUploadedFileList: [File] = []
     @Published public var isFileErrorOccured: Bool = false
     public let title = String(localized: "Attachments", bundle: .core)
+    public let subTitle: String?
     public let fileErrorTitle = String(localized: "Error", bundle: .core)
     public let fileErrorMessage = String(localized: "Failed to add attachment. Please try again!", bundle: .core)
+    @Published public var isShowingCancelDialog = false
+    public let confirmAlert = ConfirmationAlertViewModel(
+        title: String(localized: "Are your sure?", bundle: .core),
+        message: String(localized: "You have some not uploaded attachments that are not uploaded so cannot be added as attachment!", bundle: .core),
```

Comment:
Shouldn't there be some punctuation added? Plus I think maybe `...so those cannot be added...`
Does this come from figma?

Reply:
Unfortunately, there is no design for that, so it might be incorrect. I will check all of the string values with Product.